Formula/xh.rb

```diff
@@ -1,10 +1,10 @@
-class HtRust < Formula
+class Xh < Formula
   desc "Yet another HTTPie clone"
-  homepage "https://github.com/ducaale/ht"
-  url "https://github.com/ducaale/ht/archive/v0.6.0.tar.gz"
-  sha256 "a990e2973cd62cc483dc875e5ee880bfdd759e73f8e6ccf63b188092f7d9f293"
+  homepage "https://github.com/ducaale/xh"
+  url "https://github.com/ducaale/xh/archive/v0.7.0.tar.gz"
+  sha256 "428edefbb58ffdb10a731ef0f1bb15d77bb90434f3ea72b41f8e4870ac74ec49"
   license "MIT"
-  head "https://github.com/ducaale/ht.git"
+  head "https://github.com/ducaale/xh.git"
 
   bottle do
```

Comment:
You also need to remove the `bottle` block:
```
xh:
  * New formulae in homebrew/core should not have a `bottle do` block
Error: 1 problem in 1 formula detected
```

Reply:
Should I do it in a separate commit?

Reply:
Preferably squash it with your first one that renames the formula. Since you're doing that already, change the message header to `ht-rust: rename to xh`. (Commit message style in homebrew/core is to put the formula name first.) Thanks!

Reply:
Done